scripts/setup_colab.sh

```diff
@@ -0,0 +1,243 @@
+# For building glfw: cmake, xorg-dev
+# Required for mujoco_py: libglew-dev, patchelf, libosmesa6-dev
+# Required for OpenAI gym: libpq-dev, ffmpeg, libjpeg-dev, swig, libsdl2-dev
+# Required for OpenAI baselines: libopenmpi-dev, openmpi-bin
+>&2 echo "start of setup_colab.sh"
+echo "Installing garage dependencies"
+echo "You will probably be asked for your sudo password"
+sudo apt -y -q update
+sudo apt install -y \
+  wget \
+  bzip2 \
+  unzip \
+  git \
+  cmake \
+  xorg-dev \
+  libglew-dev \
+  patchelf \
+  libosmesa6-dev \
+  libpq-dev \
+  ffmpeg \
+  libjpeg-dev \
+  swig \
+  libsdl2-dev \
+  libopenmpi-dev \
+  openmpi-bin
+
+# Build GLFW because the Ubuntu 16.04 version is too old
+# See https://github.com/glfw/glfw/issues/1004
+sudo apt purge -y libglfw*
+GLFW_DIR="$(mktemp -d)/glfw"
+git clone https://github.com/glfw/glfw.git "${GLFW_DIR}"
+cd "${GLFW_DIR}"
+git checkout 0be4f3f75aebd9d24583ee86590a38e741db0904
+mkdir glfw-build
+cd glfw-build
+sudo cmake -DBUILD_SHARED_LIBS=ON -DGLFW_BUILD_EXAMPLES=OFF \
+  -DGLFW_BUILD_TESTS=OFF -DGLFW_BUILD_DOCS=OFF ..
+sudo make -j"$(nproc)"
+sudo make install
+cd "${GARAGE_DIR}"
+
+# Leave a note in ~/.bashrc for the added environment variables
+if [[ "${_arg_modify_bashrc}" = on ]]; then
+  echo -e "\n# Added by the garage installer" >> "${BASH_RC}"
+fi
+
+# Set up MuJoCo (for gym)
+if [[ ! -d "${HOME}/.mujoco/mjpro150" ]]; then
+  mkdir -p "${HOME}"/.mujoco
+  MUJOCO_ZIP="$(mktemp -d)/mujoco.zip"
+  wget https://www.roboti.us/download/mjpro150_linux.zip -O "${MUJOCO_ZIP}"
+  unzip -u "${MUJOCO_ZIP}" -d "${HOME}"/.mujoco
+else
+  print_warning "MuJoCo is already installed"
+fi
+# Set up MuJoCo 2.0 (for dm_control)
+if [[ ! -d "${HOME}/.mujoco/mujoco200_linux" ]]; then
+  mkdir -p "${HOME}"/.mujoco
+  MUJOCO_ZIP="$(mktemp -d)/mujoco.zip"
+  wget https://www.roboti.us/download/mujoco200_linux.zip -O "${MUJOCO_ZIP}"
+  unzip -u "${MUJOCO_ZIP}" -d "${HOME}"/.mujoco
+fi
+# dm_control viewer requires MUJOCO_GL to be set to work
+if [[ "${_arg_modify_bashrc}" = on ]]; then
+  echo "export MUJOCO_GL=\"glfw\"" >> "${BASH_RC}"
+fi
+# Configure MuJoCo as a shared library
+export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:${HOME}/.mujoco/mjpro150/bin"
+LD_LIB_ENV_VAR="LD_LIBRARY_PATH=\"\$LD_LIBRARY_PATH:${HOME}/.mujoco/mjpro150"
+LD_LIB_ENV_VAR="${LD_LIB_ENV_VAR}/bin\""
+if [[ "${_arg_modify_bashrc}" = on ]]; then
+  echo "export ${LD_LIB_ENV_VAR}" >> "${BASH_RC}"
+fi
+
+# We need a MuJoCo key to import mujoco_py
+cp "${_arg_mjkey}" "${HOME}/.mujoco/mjkey.txt"
+{
+  # Prevent pip from complaining about available upgrades
+  pip install --upgrade pip
+
+  # install mujoco-py, given issue 66
+  # https://github.com/openai/mujoco-py/issues/66
+  # pip install git+git://github.com/openai/mujoco-py.git@1.50.1.1 # requires mujoco activation key
+  MUJOCO_150="$(mktemp -d)/mujoco_150"
+  git clone --branch 1.50.1.1 --depth 1 https://github.com/openai/mujoco-py.git $MUJOCO_150
```

Comment:
`setup.py` inside garage already includes `mujoco-py`, so once you execute `pip install -e [all].`, that `mujoco-py` installation will override this one. Remove.